package.json

```diff
@@ -16,7 +16,8 @@
     "flow-coverage": "flow-coverage-report -t text -t html -f ./node_modules/.bin/flow -i 'packages/wonder-blocks-*/**/*.js' -x 'packages/wonder-blocks-*/dist/*.js' -x '**/*.test.js' -x '**/node_modules/**/*.js'",
     "gen-snapshot-tests": "node utils/gen-snapshot-tests.js && prettier --write packages/*/generated-snapshot.test.js",
     "gen-styleguide-prod-config": "node utils/gen-styleguide-prod-config.js && prettier --write styleguide.prod.config.js",
-    "jest": "jest",
+    "jest": "jest --config config/jest/test.config.js",
+    "test:vite": "jest --config config/jest/vite.config.js",
```

Comment:
`jest` has a `packages` option which allows you to specify multiple configurations in a single file, but I couldn't get it to work so this is what I ended up with.